app/server/routes/api.js

```diff
@@ -278,14 +278,36 @@ module.exports = function(router) {
    *   code: STRING
    * }
    */
-  router.put('/users/:id/team', isOwnerOrAdmin, function(req, res){
+
+  // Route for joining a team
+  router.post('/users/:id/team', isOwnerOrAdmin, function(req, res) {
```

Comment:
Is there a reason why you have this method? I don't thinnk this will ever be used at all since clicking the Join Team will only call on the function for **PUT /users/:id/team**